contracts/protektCore/helpers/HarvestRewardsCompoundDaiManual.sol

```diff
@@ -40,4 +45,25 @@ contract HarvestRewardsCompoundDaiManual {
         emit HarvestRewards(amount);
         // emit HarvestRewards(0);
     }
+
+    /**
+     * @dev Deposits coreTokens into Compound and returns underlyingTokens
+     */
+    function depositCoreTokens(uint256 _amount, address depositor) public returns (uint256) {
+        
+        // Bring in amount from depositer
+        IERC20(daiTokenAddress).safeTransferFrom(depositor, address(this), _amount);
+
+        // Give approval to cDai to deposit the tokens
+        IERC20(daiTokenAddress).safeApprove(cDaiTokenAddress, _amount);
+
+        // deposit to the address
+        assert(cToken(cDaiTokenAddress).mint(_amount) == 0);
```

Comment:
Is checking if `==0` a status check?

Reply:
Yeah from compound docs - just checks it worked